src/groups/bmq/bmqc/bmqc_orderedhashmapwithhistory.h

```diff
@@ -635,6 +635,21 @@ OrderedHashMapWithHistory<KEY, VALUE, HASH, VALUE_TYPE>::insert(
         }
         it->d_next = d_impl.end();
     }
+    else {
+        if (!it->d_isLive) {
+            // Corner case of (re)inserting undetected duplicate
```

Comment:
`bmqc_orderedhashmapwithhistory` is a general-purpose container, so the documentation for it should not mention one of its concrete usages from another level of application logics 
```suggestion
            // Corner case of (re)inserting an element that was previously
            // marked for deletion but still exists in history.
```